decidim-core/config/initializers/dragonfly.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require "dragonfly"
+require "avatar_magick"
+
+# Configure
+Dragonfly.app.configure do
+  plugin :imagemagick
+  plugin :avatarmagick
+
+  secret "c7e2a7239bd280c596739e13aba675994af2d570ad9e366b6166d15c2f4db4d9"
+
+  url_format "/media/:job/:name"
+
+  datastore :file,
+            root_path: Rails.root.join("public/system/dragonfly", Rails.env),
```

Comment:
Please use a cache storage instead of filesystem, as filesystem is volatile in some platforms like heroku.